.eslintrc.js

```diff
@@ -0,0 +1,155 @@
+module.exports = {
+    root: true,
+    parser: 'babel-eslint',
+    parserOptions: {
+        ecmaVersion: 2018,
+        sourceType: 'module',
+        ecmaFeatures: {
+            legacyDecorators: true
+        }
+    },
+    plugins: [
+        'ember'
+    ],
+    extends: [
+        'eslint:recommended',
+        'plugin:ember/recommended'
+    ],
+    env: {
+        browser: true
+    },
+    rules: {
+        'ember/no-jquery': 'error',
+
+        // AirBnB style
+        'array-bracket-spacing': [
```

Comment:
I'd prefer to keep this repo as close to the default blueprint as possible. Can you reset this file to the default setup?